app/src/main/java/com/door43/translationstudio/core/TargetTranslation.java

```diff
@@ -177,6 +177,27 @@ public String getTargetLanguageId() {
     }
 
     /**
+     * apply new format to translation
+     * @param translationFormat
+     */
+    public void applyFormat(TranslationFormat translationFormat) {
```

Comment:
This is mixing migration code with the object logic. For migration code it is always safest to manually manipulate files and not depend too much on the main logic in the object since a future change in the object could break a migration step. Also, we should avoid using any global variables in the migration code since changes to these in the future will likewise break the migration code.